frontend/src/components/study-card-images/study-image.tsx

```diff
@@ -0,0 +1,46 @@
+import { React, useState, useEffect } from '@common'
+import { ParticipantStudy } from '@api'
+import { getImageUrl, getAltText } from './card-images'
+import { colors } from '@theme';
+
+const SVGManipulator: React.FC<{src: string, fillColor: string, altText: string}> = ({ src, fillColor, altText }) => {
```

Comment:
I am not an expert with SVGs so im curious about this approach, it looks like we are modifying the `fill` property in the SVG and then replacing the image?

would this be possible/simpler with CSS? something like this? not sure if it work, but would simplify this a bit: https://stackoverflow.com/questions/9529300/can-i-change-the-fill-color-of-an-svg-path-with-css

Reply:
I tried that approach initially and found that since we are using an img tag and using the svg as a src, it would not work. For the above approach to work, the svg file should be rendered directly instead of using an img tag, since the images are in the public folder, I could not import them, to use them directly.

Reply:
Another problem I noticed with that approach is that on setting the fill values on the svg container, it automatically applies to all the paths, unless a fill property is already present. So we have to manually edit all the paths which don't have a fill value and set it to none, and remove the fill value for the paths which have the secondary color. This way by setting the fill value once, it is applied to all the paths which don't have a fill value (secondary color). The only problem is, the svg files have to be edited manually and should be placed inside the src folder